07-11 13:28:44 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:181)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:315)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:226)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:199)
	at com.dubture.symfony.index.SymfonyDbFactory.<init>(SymfonyDbFactory.java:127)
	at com.dubture.symfony.index.SymfonyDbFactory.getInstance(SymfonyDbFactory.java:58)
	at com.dubture.symfony.index.SymfonyIndexer.<init>(SymfonyIndexer.java:51)
	at com.dubture.symfony.index.SymfonyIndexer.getInstance(SymfonyIndexer.java:61)
	at com.dubture.symfony.core.SymfonyCorePlugin$1.resourceChanged(SymfonyCorePlugin.java:46)
	at org.eclipse.core.internal.events.NotificationManager$1.run(NotificationManager.java:291)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.NotificationManager.notify(NotificationManager.java:285)
	at org.eclipse.core.internal.events.NotificationManager.broadcastChanges(NotificationManager.java:149)
	at org.eclipse.core.internal.resources.Workspace.broadcastBuildEvent(Workspace.java:381)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:139)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 25 more
07-16 10:45:43 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:181)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:315)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:226)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:199)
	at com.dubture.symfony.index.SymfonyDbFactory.<init>(SymfonyDbFactory.java:103)
	at com.dubture.symfony.index.SymfonyDbFactory.getInstance(SymfonyDbFactory.java:58)
	at com.dubture.symfony.index.SymfonyIndexer.<init>(SymfonyIndexer.java:51)
	at com.dubture.symfony.index.SymfonyIndexer.getInstance(SymfonyIndexer.java:61)
	at com.dubture.symfony.core.SymfonyCorePlugin$1.resourceChanged(SymfonyCorePlugin.java:46)
	at org.eclipse.core.internal.events.NotificationManager$1.run(NotificationManager.java:291)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.NotificationManager.notify(NotificationManager.java:285)
	at org.eclipse.core.internal.events.NotificationManager.broadcastChanges(NotificationManager.java:149)
	at org.eclipse.core.internal.resources.Workspace.broadcastBuildEvent(Workspace.java:381)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:139)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 25 more
07-16 11:14:46 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:181)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:315)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:226)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:199)
	at com.dubture.symfony.index.SymfonyDbFactory.<init>(SymfonyDbFactory.java:103)
	at com.dubture.symfony.index.SymfonyDbFactory.getInstance(SymfonyDbFactory.java:58)
	at com.dubture.symfony.index.SymfonyIndexer.<init>(SymfonyIndexer.java:51)
	at com.dubture.symfony.index.SymfonyIndexer.getInstance(SymfonyIndexer.java:61)
	at com.dubture.symfony.core.SymfonyCorePlugin$1.resourceChanged(SymfonyCorePlugin.java:46)
	at org.eclipse.core.internal.events.NotificationManager$1.run(NotificationManager.java:291)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.NotificationManager.notify(NotificationManager.java:285)
	at org.eclipse.core.internal.events.NotificationManager.broadcastChanges(NotificationManager.java:149)
	at org.eclipse.core.internal.resources.Workspace.broadcastBuildEvent(Workspace.java:381)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:139)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 25 more
07-16 20:39:47 jdbc[2]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:181)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:315)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:226)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:199)
	at com.dubture.symfony.index.SymfonyDbFactory.<init>(SymfonyDbFactory.java:103)
	at com.dubture.symfony.index.SymfonyDbFactory.getInstance(SymfonyDbFactory.java:58)
	at com.dubture.symfony.index.SymfonyIndexer.<init>(SymfonyIndexer.java:51)
	at com.dubture.symfony.index.SymfonyIndexer.getInstance(SymfonyIndexer.java:61)
	at com.dubture.symfony.core.SymfonyCorePlugin$1.resourceChanged(SymfonyCorePlugin.java:46)
	at org.eclipse.core.internal.events.NotificationManager$1.run(NotificationManager.java:291)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.NotificationManager.notify(NotificationManager.java:285)
	at org.eclipse.core.internal.events.NotificationManager.broadcastChanges(NotificationManager.java:149)
	at org.eclipse.core.internal.resources.Workspace.broadcastBuildEvent(Workspace.java:381)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:139)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 25 more
07-17 16:39:49 jdbc[4]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.get(DbException.java:169)
	at org.h2.message.DbException.get(DbException.java:146)
	at org.h2.message.DbException.get(DbException.java:135)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:93)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:181)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:315)
	at org.h2.jdbcx.JdbcDataSource.getPooledConnection(JdbcDataSource.java:341)
	at org.h2.jdbcx.JdbcConnectionPool.getConnectionNow(JdbcConnectionPool.java:226)
	at org.h2.jdbcx.JdbcConnectionPool.getConnection(JdbcConnectionPool.java:199)
	at com.dubture.symfony.index.SymfonyDbFactory.createConnection(SymfonyDbFactory.java:181)
	at com.dubture.symfony.index.SymfonyDbFactory.getTransDao(SymfonyDbFactory.java:200)
	at com.dubture.symfony.index.SymfonyIndexer.<init>(SymfonyIndexer.java:54)
	at com.dubture.symfony.index.SymfonyIndexer.getInstance(SymfonyIndexer.java:61)
	at com.dubture.symfony.core.model.SymfonyModelAccess.<init>(SymfonyModelAccess.java:94)
	at com.dubture.symfony.core.model.SymfonyModelAccess.getDefault(SymfonyModelAccess.java:103)
	at com.dubture.symfony.core.compiler.SourceElementRequestor.<init>(SourceElementRequestor.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at java.lang.Class.newInstance0(Unknown Source)
	at java.lang.Class.newInstance(Unknown Source)
	at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:184)
	at org.eclipse.core.internal.registry.ExtensionRegistry.createExecutableExtension(ExtensionRegistry.java:905)
	at org.eclipse.core.internal.registry.ConfigurationElement.createExecutableExtension(ConfigurationElement.java:243)
	at org.eclipse.core.internal.registry.ConfigurationElementHandle.createExecutableExtension(ConfigurationElementHandle.java:55)
	at org.eclipse.php.internal.core.compiler.PHPSourceElementRequestor.<init>(PHPSourceElementRequestor.java:97)
	at org.eclipse.php.internal.core.compiler.ast.parser.PHPSourceElementParser.createVisitor(PHPSourceElementParser.java:30)
	at org.eclipse.dltk.core.AbstractSourceElementParser.parseSourceModule(AbstractSourceElementParser.java:28)
	at org.eclipse.php.internal.core.compiler.ast.parser.PHPSourceElementParser.parseSourceModule(PHPSourceElementParser.java:26)
	at org.eclipse.dltk.internal.core.AbstractSourceModule.buildStructure(AbstractSourceModule.java:507)
	at org.eclipse.dltk.internal.core.Openable.generateInfos(Openable.java:195)
	at org.eclipse.dltk.internal.core.ModelElement.openWhenClosed(ModelElement.java:196)
	at org.eclipse.dltk.internal.core.ModelElement.getElementInfo(ModelElement.java:183)
	at org.eclipse.dltk.internal.core.ModelElement.getChildren(ModelElement.java:368)
	at org.eclipse.dltk.internal.core.ModelElement.getChildrenOfType(ModelElement.java:347)
	at org.eclipse.dltk.internal.core.ModelElement.getChildrenOfType(ModelElement.java:342)
	at org.eclipse.dltk.internal.core.AbstractSourceModule.getTypes(AbstractSourceModule.java:357)
	at org.eclipse.dltk.internal.core.AbstractSourceModule.getAllTypes(AbstractSourceModule.java:138)
	at com.zend.php.semanticanalysis.internal.core.checks.DuplicateClassMethodCheck.check(DuplicateClassMethodCheck.java:27)
	at com.zend.php.semanticanalysis.internal.core.SemanticAnalysisBuilder.buildModelElements(SemanticAnalysisBuilder.java:128)
	at com.zend.php.semanticanalysis.internal.core.SemanticAnalysisBuilder.build(SemanticAnalysisBuilder.java:211)
	at org.eclipse.dltk.internal.core.builder.ScriptBuilder.fullBuild(ScriptBuilder.java:448)
	at org.eclipse.dltk.internal.core.builder.ScriptBuilder.build(ScriptBuilder.java:156)
	at org.eclipse.core.internal.events.BuildManager$2.run(BuildManager.java:726)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:199)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:239)
	at org.eclipse.core.internal.events.BuildManager$1.run(BuildManager.java:292)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.core.internal.events.BuildManager.basicBuild(BuildManager.java:295)
	at org.eclipse.core.internal.events.BuildManager.basicBuildLoop(BuildManager.java:351)
	at org.eclipse.core.internal.events.BuildManager.build(BuildManager.java:374)
	at org.eclipse.core.internal.events.AutoBuildJob.doBuild(AutoBuildJob.java:143)
	at org.eclipse.core.internal.events.AutoBuildJob.run(AutoBuildJob.java:241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-169]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:329)
	... 59 more
